ts/screens/authentication/cie/CiePinScreen.tsx

```diff
@@ -91,26 +91,37 @@ class CiePinScreen extends React.PureComponent<Props, State> {
 
   public render() {
     return (
-      <TopScreenComponent
+      <BaseScreenComponent
         goBack={true}
-        headerTitle={I18n.t("authentication.cie.pin.pinCardHeader")}
+        headerTitle={I18n.t("authentication.cie.pin.header")}
       >
-        <ScrollView>
+        <Content>
           <ScreenContentHeader
-            title={I18n.t("authentication.cie.pin.pinCardTitle")}
-            icon={require("../../../../img/icons/icon_insert_cie_pin.png")}
+            padded={false}
+            title={I18n.t("authentication.cie.pin.title")}
           />
           <View spacer={true} />
-          <View style={styles.container}>
-            <CiePinpad
-              pin={this.state.pin}
-              pinLength={CIE_PIN_LENGTH}
-              description={I18n.t("authentication.cie.pin.pinCardContent")}
-              onPinChanged={this.handelOnPinChanged}
-              onSubmit={this.handleOnContinue}
-            />
-          </View>
-        </ScrollView>
+          <CiePinpad
+            pin={this.state.pin}
+            pinLength={CIE_PIN_LENGTH}
+            onPinChanged={this.handelOnPinChanged}
+            onSubmit={this.handleOnContinue}
+          />
+          <Text style={styles.reducedHeight}>
```

Comment:
```suggestion
          <View>
            <Text bold={true}>
              {I18n.t("authentication.cie.pin.description.block1")}
            </Text>
            <Text>{`${I18n.t(
              "authentication.cie.pin.description.block2"
            )}\n${I18n.t(
              "authentication.cie.pin.description.block3"
            )}\n${I18n.t("authentication.cie.pin.description.block4")}`}</Text>
          </View>
```
Since the default flex direction is row, could it be the same?